lib/ffmpeg/transcoderbase.py

```diff
@@ -205,7 +205,12 @@ def transcode(self):
           "{}_{}_{}.yuv".format(self.case, n, channel))
         osyuv = filepath2os(yuv)
         vppscale = self.get_vpp_scale(self.width, self.height, "sw")
+        codec = output["codec"]
+        mode = output["mode"]
+        encoder = self.get_encoder(codec, mode)
         iopts = "-i {}"
+        if ("av1_qsv") == encoder:
+          iopts = "-c:v av1_qsv -i {}" 
```

Comment:
Maybe it would be better if we refactor some logic so we can specialize this in lib/ffmpeg/qsv/transcoder.py instead.